src/main/java/eu/decentsoftware/holograms/api/utils/items/SkullUtils.java

```diff
@@ -118,17 +120,25 @@ public static String getTexture(@NonNull ItemStack itemStack) {
 	 * Set the Base64 texture of the given skull ItemStack.
 	 *
 	 * @param itemStack The ItemStack.
-	 * @param texture   The new skull texture (Base64).
+	 * @param texture   The new skull texture (Base64 or URL).
 	 */
 	public static void setSkullTexture(@NonNull ItemStack itemStack, @NonNull String texture) {
 		try {
 			ItemMeta meta = itemStack.getItemMeta();
 			if (meta instanceof SkullMeta) {
 				GameProfile profile = new GameProfile(UUID.randomUUID(), "");
-				Property property = new Property("textures", texture);
 
-				PropertyMap properties = profile.getProperties();
-				properties.put("textures", property);
+				String json = Base64.getDecoder().decode(texture);
+				if(!json.contains("\"url\":\"")){
+					// Decoded json does not contain a skin URL, so check if the 'texture' argument is a URL.
+					if(texture.startsWith("http://")); // We are all set.
```

Comment:
That is problematic as it ignores the possibility of a `https://` url. So either also check if it starts with `https://` or maybe just check if it starts with `http` (Tho that could be a bit to vague)

Reply:
Fair point, but Minecraft's skin server only uses http:// (unencrypted skinfiles). They could always change this at a future date, but since the game won't render skin files from outside Minecraft's servers, I didn't bother for https:// (since no valid skin will start with that).

The old HD skins (super-high resolution heads, 640x640 etc) from the Education Edition website used https://, but when Minecraft removed that they did so on their server end, so it applies to ALL versions of Minecraft

If you want to be flexibly in case they start re-allowing that in the future, could do "http" as you said. It's a bit more vague but likely not an issue

Reply:
What I also find a bit problematic is, how this setup could allow any URL to work as a skin. While I doubt it will work, do I feel like it's not a good idea to leave it up to the server to do the cleanup here...

It just feels... odd.

Reply:
Fair enough, I'll update the check to be more thorough (by checking for the full Minecraft skin server URL)